README.md

```diff
@@ -19,6 +18,26 @@ But when you need to scale your application Caddy is limited to its static confi
 To overcome this issue we are using `docker-gen` to generate configuration everytime a container spawns or dies.
 Now scaling is easy!
 
+## CADDY 2
+
+BREAKING CHANGES HERE!
+
+Options to configure:
+
+- `virtual.host` domain name, don't pass `http://` or `https://`, you can separate them with space,
+- `virtual.alias` domain alias, e.q. `www` prefix,
+- `virtual.port` port exposed by container, e.g. `3000` for React apps in development,
+- `virtual.tls-email` the email address to use for the ACME account managing the site's certificates,
+- `virtual.auth.username` and
+- `virtual.auth.password` together provide HTTP basic authentication.
+
+Password should be a string `base64` encoded from `bcrypt` hash. You can use https://bcrypt-generator.com/ with default config and https://www.base64encode.org/.
+
+## Backing up certificates
+
+To backup certificates make a volume:
+
+`./caddy-info:/data/caddy`
```

Comment:
Let's add a bit more context here (with yaml like this):

```yml
services:
  caddy:
    volumes:
      - ./caddy-info:/data/caddy
```

Reply:
:heavy_check_mark: